app/game/field.py

```diff
@@ -216,17 +220,39 @@ def check_obstacle(self, point):
             return False
         return True
 
+    def get_field_object(self, point):
+        if field_object(self.objects, point) is Object.wall:
+            return Object.wall
+
+        if field_object(self.objects, point) is Object.block:
+            return Object.block
+
+        if field_object(self.bombs, point) is not None:
+            return self.bombs[point.x][point.y]
+
+        if field_object(self.items, point) is not None:
+            return self.items[point.x][point.y]
+
+        for person in self.persons:
+            if person.point.x is point.x and person.point.y is point.y:
```

Comment:
is judge identity of object. 
In python, small int object is reusable so 1 is 1 and 1 is (1+1-1) are also True, but big number is not so.
For example, 12345678901234567890 is (12345678901234567890+1-1) is False.
You can use == for judging equality for variable.